docs/learn/dapp-developer/connect-profile.md

```diff
@@ -16,6 +16,12 @@ Optionally you can ⚒️ [deploy a Universal Profile programmatically](../../le
 
 :::
 
+:::note
+
+If you expect your users to have MetaMask or other similar browser extensions installed, you can also connect to the Universal Profile Browser Extension directly, using the `window.lukso` object.
```

Comment:
Maybe you can put a note here, about the `window.lukso` object is... e.g. that its an clone of the `ethereum` object with all the functionality of the [Ethereum JSON RPC](https://ethereum.org/en/developers/docs/apis/json-rpc/)